### PR TITLE
OBSDOCS-562 - Logging Features by version, preliminary draft.

### PR DESCRIPTION
**Pull request by libander**
Version(s):

Issue:

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: _topic_maps/_topic_map.yml [hunk 1]
```diff
@@ -2560,6 +2560,8 @@ Topics:
     File: logging-5-7-release-notes
 - Name: Support
   File: cluster-logging-support
+- Name: Getting started with Logging
```
Comment (abrennan89) [inline _topic_maps/_topic_map.yml:2563]: IMHO this should go after the About section, i.e. first what is logging, second, how do I use it?

File: logging/logging-getting-started.adoc [hunk 2]
```diff
@@ -0,0 +1,58 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="logging-getting-started"]
+= Getting started with {logging}
+include::_attributes/common-attributes.adoc[]
+:context: <logging-getting-started>
```
Comment (abrennan89) [inline logging/logging-getting-started.adoc:5]:
```suggestion
:context: logging-getting-started
```
preview link is broken and I wonder if this is why - also, include the attributes above the title if you're using an attribute in the title

File: logging/logging-getting-started.adoc [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+.Cluster Logging Operator
+* Flow control/rate limiting
+* Audit Log Filtering
+* Cluster Logging HTTP Server
+
+.Vector collector
+* Compliance with TLS Security profile configuration for server endpoints
+* Support for TLS Security profiles that matches the output
+* Multi ClusterLogForwarder
```
Comment (abrennan89) [inline logging/logging-getting-started.adoc:40]: Please use the attributes for the Operator names. If you need to update the {clo} one that's fine, but I want us to stay in the habit of using the attributes for consistency.

Comment (abrennan89) [reply logging/logging-getting-started.adoc:40]: All of these points should be sentence case as well

**Review by abrennan89 — COMMENTED** [review]
Some initial comments. I like the direction, though.

File: _topic_maps/_topic_map.yml (hunk 1 above)
Comment (libander) [reply _topic_maps/_topic_map.yml:2563]: I think you're right, good call!

File: logging/logging-getting-started.adoc (hunk 3 above)
Comment (libander) [reply logging/logging-getting-started.adoc:40]: Oh this isn't remotely peer-review ready, I was just compiling the initial list & figuring out presentation form. Most of those are copy-pasted from the enablement slides.  :)

File: logging/logging-getting-started.adoc (hunk 2 above)
Comment (libander) [reply logging/logging-getting-started.adoc:5]: I was scratching my head on that one, thanks!

**Pull request closed without merging**